contracts/modules/cover/CoverNFT.sol

```diff
@@ -0,0 +1,24 @@
+
+import "@openzeppelin/contracts-v4/token/ERC721/ERC721.sol";
+import "../../interfaces/ICover.sol";
+
+contract CoverNFT is ERC721 {
+
+  ICover cover;
```

Comment:
`cover` is a state variable and no visibility is specified. There's no way to modify it so there's no point in having it in storage - let's make it immutable (cover is a proxy anyway) and let's make it public.